src/xc_integrator/replicated/host/reference_replicated_xc_host_integrator_exc_grad.hpp

```diff
@@ -172,13 +212,47 @@ void ReferenceReplicatedXCHostIntegrator<ValueType>::
       d2basis_zz_eval = d2basis_yz_eval + npts * nbe;
     }
 
+    if( func.is_mgga() ) {
+      d2basis_xx_eval = dbasis_z_eval   + npts * nbe;
+      d2basis_xy_eval = d2basis_xx_eval + npts * nbe;
+      d2basis_xz_eval = d2basis_xy_eval + npts * nbe;
+      d2basis_yy_eval = d2basis_xz_eval + npts * nbe;
+      d2basis_yz_eval = d2basis_yy_eval + npts * nbe;
+      d2basis_zz_eval = d2basis_yz_eval + npts * nbe;
+      if ( true ) {
```

Comment:
This is the laplacian check right? I would actually store this somewhere and just set it to true (with a comment to update once the lapl requirement is checkable)